processor/geoipprocessor/config.go

```diff
@@ -3,9 +3,21 @@
 
 package geoipprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/geoipprocessor"
 
+import "errors"
+
 // Config holds the configuration for the GeoIP processor.
-type Config struct{}
+type Config struct {
+	// Fields defines a list of resource attributes keys to look for the IP value.
+	// The first matching attribute will be used to gather the IP geographical metadata.
+	// Example:
+	//   fields:
+	//     - client.ip
+	Fields []string `mapstructure:"fields"`
```

Comment:
`fields` is not an OTel term to describe resource attributes. Perhaps `resource_attributes` or just `resource` would be better? :thinking: 

Alternatively, we could deliver the first iteration of the processor without exposing this configuration at all, and instead look at a standard attribute like [source.address](https://github.com/open-telemetry/semantic-conventions/blob/v1.26.0/docs/general/attributes.md#source)?

Reply:
I like the idea of using the standard `source.address` attribute to look for the IP in this initial implementation. That way, we still don't expose any configuration to the user, and we can further discuss whether to use `fields`, `resource`, `resource_attributes`, etc.

Reply:
Configuration option removed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33319/commits/33ab2f5deb72d79fb21e985e200aeb93c3be67c0